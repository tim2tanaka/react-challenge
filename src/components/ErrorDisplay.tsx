import { useAppState } from '../app_state/context';

export function ErrorDisplay() {
  const state = useAppState();
  return (
    <div>
      <span className="error-msg"> {state.errorMsg} </span>
    </div>
  );
}
