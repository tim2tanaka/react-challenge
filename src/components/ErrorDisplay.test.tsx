import { render } from '@testing-library/react';
import { ErrorDisplay } from './ErrorDisplay';

describe('ErrorDisplay', () => {
  test('render ErrorDisplay', () => {
    const { container } = render(<ErrorDisplay />);
    expect(container).toMatchSnapshot();
  });
});
