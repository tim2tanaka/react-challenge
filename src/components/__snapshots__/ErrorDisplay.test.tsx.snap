// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorDisplay render ErrorDisplay 1`] = `
<div>
  <div>
    <span
      class="error-msg"
    >
       
       
    </span>
  </div>
</div>
`;
